Week1/index.js

```diff
@@ -0,0 +1,245 @@
+'use-strict';
```

Comment:
So I should no read this file right ?
you can use git rm Week1/index.js then commit and push to remove it .